messages.go

```diff
@@ -58,12 +58,33 @@ func (rId RoundID) String() string {
 	return uuid.UUID(rId).String()
 }
 
+// Equal returns true if and only if the given RoundID equals rId.
```

Comment:
I think we asked you to mis-correct this one: we have an argument, `rID2`, and a method on a structure `RoundID`. So I think the correct comment would be:

```
// Equal returns true if rID2 equals this RoundID
```

and also for the other `Equals`. This way we also describe the argument of the method instead of referencing it only as `given RoundID`.